.github/workflows/pr.yaml

```diff
@@ -54,7 +54,32 @@ jobs:
   audit:
     name: Audit
     runs-on: ubuntu-latest
-    if: false
+    steps:
+      - uses: actions/checkout@v3
+      - uses: FuelLabs/github-actions/setups/node@master
+        with:
+          node-version: 20.11.0
+          pnpm-version: 9.5.0
+      - name: Install jq
+        run: sudo apt-get install jq
+      - run: |
+          pnpm audit --prod --json | jq '
```

Comment:
I believe the best way to go is to turn audit off for a couple of days until they update their databases with the safe @solana/web3.js version... Maybe is safer then include 20 lines of javascript on the workflow...

Reply:
These 20 lines of bash script are pretty simple.. it just converts the audit output to json and then analyzes it. This same script has worked flawlessly in [fuel-wallet](https://github.com/FuelLabs/fuels-wallet/blob/21eb5cd8699002d7d46c57432f6c9e3cfa89ea03/.github/workflows/pr.yaml#L68) for 4 months now.